aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -166,6 +167,9 @@ def save(
                 channel_colors=channel_colors,
                 dimension_order=dimension_order,
             )
+        # if it is data from CZI->OME via XSLT then
+        elif type(ome_xml) is ET.Element:
+            self.omeMetadata = ome_xml
```

Comment:
You should update the docstring for this writer to tell people that `ET.Element` is also now supported

Reply:
Also because this is an `elif` it will never run. The above `if` will always run instead. Need to make the above `if` check be an `isinstance` check. Same with this. `if isinstance(ome_xsl, ET.Element)`

Reply:
Thanks, ugg my bad.

Reply:
You are correct. I thought it said "if x is not None" for some reason. That said I would still change this check to be `if isinstance()`